Directory.Build.props

```diff
@@ -44,34 +49,34 @@
   <Choose>
     <When Condition="'$(Configuration)' == 'Debug' and '$(IsDesignProject)' != 'true'">
       <!-- Debug builds have this turned on by default, but it breaks our Xaml Islands Scenarios -->
-      <PropertyGroup>
-        <EnableTypeInfoReflection>false</EnableTypeInfoReflection>
+      <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
```

Comment:
This is already checked two lines above here.
```suggestion
      <PropertyGroup>
```

Reply:
Must've been a mistake during rebase. Nice find. Thanks!